protocol/avdl/stellar1/remote.avdl

```diff
@@ -4,7 +4,8 @@
 
   @typedef("string") record AccountID {}
   @typedef("string") record TransactionID {}
-  @typedef("bytes")  record KeybaseTransactionID {}
+  @typedef("string") record KeybaseTransactionID {}
+  @typedef("long")   record Time {} // ms since epoch
```

Comment:
TimeMs?

Reply:
ok.  chat1.Time is same, but TimeMs clearer.

Reply:
I've spent lots of time re-figuring out the chat1 and gregor1 times. This will hopefully save the trouble